static-site/app/sars-cov-2/[[...sars-cov-2]]/content.tsx

```diff
@@ -0,0 +1,354 @@
+  const title = metadata.title as string; // eslint-disable-line @typescript-eslint/consistent-type-assertions
+
+  // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+  const datasets = (sarscov2Catalogue as { datasets: DatasetType[] })[
+    "datasets"
+  ];
+
+  const parsedDatasets = _parseDatasetsFilterList(datasets);
+
+  return (
+    <>
+      <HugeSpacer />
+      <HugeSpacer />
+
+      <h1>{title}</h1>
+
+      <SmallSpacer />
+
+      <FlexCenter>
+        <FocusParagraphCentered>
+          Labs from all over the world are sequencing SARS-CoV-2 and sharing
+          genomic data. The Nextstrain team analyzes these data on a global and
+          continental level. More specific analyses are often performed by
+          public health and academic groups from around the world. This page
+          lists a sample of such analyses. In addition to exploring SARS-CoV-2
+          evolution in finished analyses, you can use Nextclade to compare your
+          sequences to the SARS-CoV-2 reference genome, assign them to clades,
+          and see where they fall on a SARS-CoV-2 phylogeny.
+        </FocusParagraphCentered>
+      </FlexCenter>
+
+      <MediumSpacer />
+
+      <ResourceListing />
+
+      <div id="datasets">
+        <HugeSpacer count={2} />
+        <h2 className="left">All SARS-CoV-2 datasets</h2>
+        <SmallSpacer />
+        <FocusParagraph>
+          This section is an index of public Nextstrain datasets for SARS-CoV-2,
+          organized by geography. Some of these datasets are maintained by the
+          Nextstrain team and others are maintained by independent research
+          groups. If you know of a dataset not listed here, please let us know!
+          Please note that inclusion on this list does not indicate an
+          endorsement by the Nextstrain team.
+        </FocusParagraph>
+        <HugeSpacer />
+        <DatasetSelect
+          datasets={parsedDatasets}
+          columns={tableColumns}
+          interfaces={[
+            DatasetMap,
+            "FilterSelect",
+            "FilterDisplay",
+            "ListDatasets",
+          ]}
+        />
+      </div>
+
+      <div id="sit-reps">
+        <HugeSpacer count={2} />
+
+        <h2 className="left">All SARS-CoV-2 situation reports</h2>
+
+        <SmallSpacer />
+
+        <FocusParagraph>
+          We have been writing interactive situation reports using{" "}
+          <a href="https://nextstrain.github.io/auspice/narratives/introduction">
+            Nextstrain Narratives{" "}
+          </a>
+          to communicate how COVID-19 is moving around the world and spreading
+          locally. These are kindly translated into a number of different
+          languages by volunteers and Google-provided translators — click on any
+          language below to see the list of situation reports available.
+        </FocusParagraph>
+
+        <div className="row">
+          <MediumSpacer />
```

Comment:
The `<MediumSpacer />` doesn't add any spacing here because it doesn't have any width/flex. 

I suggest moving it out of the row div or wrapping it with a `<div className="col-12">`

Reply:
Will relocate outside the row div.